webapp/src/css/icon.less

```diff
@@ -81,3 +93,161 @@
     transform: rotate(360deg);
   }
 }
+
+.fa-comments:before{
```

Comment:
You can [use parametric mixins](https://lesscss.org/features/#mixins-feature-mixins-parametric-feature) that helps you reusing code, the mixin can accept the icon url or colors as parameters. We have done this before in [variables.less](https://github.com/medic/cht-core/blob/master/webapp/src/css/variables.less)

Reply:
yeah, good idea, it's done.